src/tests/utils/enum.utils.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Theme } from "$lib/types/theme";
+import { enumFromStringExists } from "$lib/utils/enum.utils";
+
+describe("enum-utils", () => {
+  it("should return enum exists", () => {
+    expect(
+      enumFromStringExists<Theme>({
```

Comment:
I guess `<typeof Theme>` would be a more precise type.

Reply:
Nice 👍